src/api/helpers/map-CIS-countries/map-CIS-country/is-high-risk-country/index.test.ts

```diff
@@ -0,0 +1,43 @@
+import { RiskClassifications } from '../../../../types';
+import { EXTERNAL_API_DEFINITIONS } from '../../../../constants';
+import isHighRiskCountry from './index';
+
+const {
+  CIS: { ESRA_CLASSIFICATION },
+} = EXTERNAL_API_DEFINITIONS;
+
+const assertions = [
+  {
+    response: false,
+    risk: null,
+  },
+  {
+    response: false,
+    risk: ESRA_CLASSIFICATION.NONE as RiskClassifications,
+  },
+  {
+    response: false,
+    risk: ESRA_CLASSIFICATION.STANDARD as RiskClassifications,
+  },
+  {
+    response: true,
+    risk: ESRA_CLASSIFICATION.HIGH as RiskClassifications,
+  },
+  {
+    response: false,
+    risk: ESRA_CLASSIFICATION.VERY_HIGH as RiskClassifications,
+  },
+];
+
+describe('isHighRiskCountry', () => {
```

Comment:
Can we match the naming of other tests?

```suggestion
describe('helpers/map-CIS-countries/map-CIS-country/is-high-risk-country, () => {
```